components/meta.js

```diff
@@ -13,20 +13,21 @@ export default function Meta() {
         rel="icon"
         type="image/png"
         sizes="32x32"
-        href="/favicon/favicon-32x32.png"
+        href="/favicon/favicon_brico_32x32.png"
```

Comment:
maybe just keep the name but replace the existing files?

either way, looks like there are a few more favicon files to replace in the public/favicon folder.